src/Symfony/Component/Console/Output/AnsiColorCode.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Output;
+
+/**
+ * @author Julien Boudry <julien@condorcet.vote>
+ */
+enum AnsiColorCode
```

Comment:
This is more a color mode than a color code.

Reply:
Maybe Enum can be renamed to AnsiColorMode, because indeed its call is relative to a Terminal, but the `method convertFromHexToAnsiColorCode()` should remain unchanged. What do you think?